python/arcticdb/version_store/_store.py

```diff
@@ -1713,6 +1721,7 @@ def compact_incomplete(
         via_iteration: Optional[bool] = True,
         sparsify: Optional[bool] = False,
         metadata: Optional[Any] = None,
+        prune_previous_version: Optional[bool] = None,
```

Comment:
Don't we need the same change in `library.py`?

Reply:
On a related note: should the new `sort_merge` function prune as well?